packages/backend-common/src/database/DatabaseManager.ts

```diff
@@ -193,6 +193,13 @@ export class DatabaseManager {
     );
   }
 
+  private getAuthConfig(pluginId: string): string | undefined {
```

Comment:
The return type here isn't really string, is it?

Seems to me that you want to return something like `undefined | true | Config`, and then use the Config interface to properly read the values out of that if the contents were the complex type. Don't leave the Config interface prematurely; you'll lose type safety and error context information etc.

Reply:
Fixed